cli/githubutil.py

```diff
@@ -140,3 +141,101 @@ def release_note_blocks_cli(
         repository_branch=repository_branch,
         commit_range=commit_range
     ).release_note_blocks()
+
+
+def cleanup_draft_releases(
+    github_cfg_name: str,
+    github_repository_owner: str,
+    github_repository_name: str,
+    latest_release_version: str,
+    dry_run: bool=True,
+):
+    github_cfg = ctx().cfg_factory().github(github_cfg_name)
+
+    githubrepobranch = GitHubRepoBranch(
+        github_config=github_cfg,
+        repo_owner=github_repository_owner,
+        repo_name=github_repository_name,
+        branch='master',
+    )
+
+    helper = GitHubRepositoryHelper.from_githubrepobranch(
+        githubrepobranch=githubrepobranch,
+    )
+    # prepare semver for later comparison
+    latest_semver = semver.parse_version_info(latest_release_version)
```

Comment:
greatest. latest implies chronological order (which is not the implemented semantics)